addons/account/wizard/accrued_orders.py

```diff
@@ -207,6 +207,8 @@ def _ellipsis(string, size):
             'date': self.date,
             'line_ids': move_lines,
         }
+        if len(orders) == 1 and self.user_has_groups('base.group_multi_currency'):
```

Comment:
maybe instead of multi currency we could check if the currency is different than company, eg 

```suggestion
        if len(orders) == 1 and orders.company_id.currency_id != orders.currency_id:
```